src/decorators/resolved-view.js

```diff
@@ -1,11 +1,15 @@
 import {Container} from 'aurelia-dependency-injection';
 import {ViewManager} from './../view-manager';
+import {useView} from 'aurelia-templating';
 
 export function resolvedView(namespace, view) {
   return function resolvedViewDecorator(target) {
     let viewManager        = Container.instance.get(ViewManager);
-    target.getViewStrategy = viewManager.resolve(namespace, view);
 
-    return target; /* handy for testing */
+    target.prototype.getViewStrategy = viewManager.resolve.bind(viewManager, namespace, view);
+
+    useView(viewManager.resolve(namespace, view))(target);
```

Comment:
Why not simply do:

```js
import {Container} from 'aurelia-dependency-injection';
import {ViewManager} from './../view-manager';
import {RelativeViewStrategy, useViewStrategy} from 'aurelia-templating';

export function resolvedView(namespace, view) {
  let viewManager = Container.instance.get(ViewManager);

  return useViewStrategy(new RelativeViewStrategy(viewManager.resolve(namespace, view)));
}
```

The only thing I'm not sure about in my example, is if RelativeViewStrategy is actually exported from `'aurelia-templating'` or `'aurelia-templating/view-strategy'`.